### DISCUSSION TITLE
Question regarding Freehandcutplotter.start() functionality

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#695] [Q&A] [answered]
Hi Marco,

Got a couple of questions regarding `show()` functionality for the `FreeHandCutPlotter()` class that you have in the library. 
1.
For my project, I made an extended class of this that had very similar functionality to the original when it came to showing the render window. When `start()` is called, the window is open and it stays open waiting for user input of some sort.

Now that I have returned to it, I find that the class I have created based on `FreeHandCutPlotter()` does not stay open when `start()` is called: it _immediately_ closes. 

I was wondering whether something has changed in the code base in the last version that may affect this behaviour? What exactly is it that keeps the window open when `start()` is called? because in `start()` no `interactive` flag is used:

```
    def start(self, *args, **kwargs):
        """Start window interaction (with mouse and keyboard)"""
        acts = [self.txt2d, self.mesh, self.points, self.spline, self.jline]
        self.show(acts + list(args), **kwargs)
        return self
```
Maybe if I understand this, I can work around or found a mistake I made somewhere...

2.
What is the keybind to bring up the list of button functions when using a `Plotter()` object? I am finding that some of the keybinds I am detecting in my class are also causing the window to spontaneously close when they are pressed. I want to make sure I am not using something that is already designated in the `Plotter()` class.

Cheers in advance, hope you're having a good `day,`
j

**Comment by marcomusy** [comment, marked as answer]
Hi J,
1. it might have changed, but i'm not 100% sure why... anyways whenever you deploy your application you should make sure it depends on a specific version of vedo... Remember that it inherits from `Plotter` so you can make it interactive with a call `freh_plt.start().interactive()`
2. you can disable all default key binds with `settings.enableDefaultKeyboardCallbacks = False` and mouse ones with `enableDefaultMouseCallbacks = False`
